src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -141,20 +147,10 @@ def spring_cloud_update(cmd, client, resource_group, name, app_insights_key=None
             trace_properties = update_tracing_config(cmd, resource_group, name, location,
                                                      app_insights_key, app_insights, (disable_distributed_tracing or disable_app_insights))
         if trace_properties is not None:
-            sdk_no_wait(no_wait, client.monitoring_settings.update_put,
-                        resource_group_name=resource_group, service_name=name, properties=trace_properties)
-
-    # update service tags
-    if tags is not None:
-        updated_resource.tags = tags
-        update_service_tags = True
-
-    if update_service_tags is False and update_service_sku is False:
-        return resource
```

Comment:
Why is this logic no longer needed?  Will deleting this logic cause breaking change to users?

Reply:
Good catch, actually, they are the duplicate code

Reply:
OK